apis/conversion/src/index.js

```diff
@@ -0,0 +1,33 @@
+import hypercube from './hypercube';
+
+const getType = async ({ halo, name, version }) => {
+  const { types } = halo;
+  const SN = await types
+    .get({
+      name,
+      version,
+    })
+    .supernova();
+  return SN;
+};
+
+const convertTo = async ({ halo, model, cellRef, newType }) => {
+  const propertyTree = await model.getFullPropertyTree();
+  const sourceQae = cellRef.current.getQae();
+  if (!sourceQae.exportProperties) {
+    throw new Error(
+      `Converting from ${propertyTree.qProperty.visualization} is not supported since there is no exportProperties method.`
+    );
```

Comment:
Need to figure out how to handle this.
Nebula should fallback to default conversion if they are missing from the qae, so the question is how we go about disabling them. Setting them to a falsy value is one way, but it isn't very explicit. Currently I'm not sure if there are any other extension level settings.
The "support" thing we use in the client is tied to the ext() file, so that is no good. Maybe there is some kind of check for export/snapshot somewhere in nebula?

Reply:
This will be handled in a separate PR, since this PR is too big already.